mcmc/StatisticalUtils.h

```diff
@@ -0,0 +1,248 @@
+#pragma once
+
+// C++ includes
+#include <iostream>
+#include <sstream>
+#include <iomanip>
+#include <vector>
+#include <algorithm>
+#include <cmath>
+
+//MaCh3 includes
+#include "samplePDF/Structs.h"
+#include "manager/manager.h"
+
+// **************************
+/// @brief  KS: Following H. Jeffreys. The theory of probability. UOP Oxford, 1998. DOI: 10.2307/3619118.
+inline std::string GetJeffreysScale(const double BayesFactor){
+// **************************
+  std::string JeffreysScale = "";
+  //KS: Get fancy Jeffreys Scale as I am to lazy to look into table every time
+  if(BayesFactor < 0)        JeffreysScale = "Negative";
+  else if( 5 > BayesFactor)  JeffreysScale = "Barely worth mentioning";
+  else if( 10 > BayesFactor) JeffreysScale = "Substantial";
+  else if( 15 > BayesFactor) JeffreysScale = "Strong";
+  else if( 20 > BayesFactor) JeffreysScale = "Very strong";
+  else JeffreysScale = "Decisive";
+
+  return JeffreysScale;
+}
+
+
+// **************************
+/// @brief  KS: Based on Table 1 in https://www.t2k.org/docs/technotes/435
+inline std::string GetDunneKaboth(const double BayesFactor){
+// **************************
+  std::string DunneKaboth = "";
+  //KS: Get fancy DunneKaboth Scale as I am to lazy to look into table every time
+
+  if(2.125 > BayesFactor)    DunneKaboth = "< 1 #sigma";
+  else if( 20.74 > BayesFactor)  DunneKaboth = "> 1 #sigma";
+  else if( 369.4 > BayesFactor) DunneKaboth = "> 2 #sigma";
+  else if( 15800 > BayesFactor) DunneKaboth = "> 3 #sigma";
+  else if( 1745000 > BayesFactor) DunneKaboth = "> 4 #sigma";
+  else DunneKaboth = "> 5 #sigma";
+
+  return DunneKaboth;
+}
+
+// *********************
+/// @brief KS: Convert sigma from normal distribution into percentage
+inline double GetSigmaValue(const int sigma) {
+// *********************
+  double width = 0;
+  switch (std::abs(sigma))
+  {
+    case 1:
+      width = 0.682689492137;
+      break;
+    case 2:
+      width = 0.954499736104;
+      break;
+    case 3:
+      width = 0.997300203937;
+      break;
+    case 4:
+      width = 0.999936657516;
+      break;
+    case 5:
+      width = 0.999999426697;
+      break;
+    case 6:
+      width = 0.999999998027;
+      break;
+    default:
+      MACH3LOG_ERROR("{}  is unsupported value of sigma", sigma);
```

Comment:
one day, somebody will find this error!

Reply:
At what point is it easier to just calculate these values